testing/environment/docker/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2.3'
+services:
+  kafka:
+    build: /Users/robbavey/code/elastic-agent-shipper/testing/environment/docker/dockerfiles/kafka 
```

Comment:
Oh, missed this initially, it looks like this file needs to have the home directory stripped out or maybe just be deleted if you were just using it for testing the prototype

Reply:
🤦‍♂️  Done